<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInfoUI</class>
 <widget class="QWidget" name="UserInfoUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户个人信息</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <widget class="QWidget" name="Headwidget" native="true">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/icon/defaultheadpicture.jpg);</string>
      </property>
     </widget>
     <widget class="QLabel" name="UserNamelabel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 12pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>昵称：</string>
      </property>
     </widget>
     <widget class="QLabel" name="UserIDlabel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 12pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwdlineEdit">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>150</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 12pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>密码：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="namelineEdit">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
